.changeset/config.json

```diff
@@ -5,7 +5,7 @@
   "fixed": [],
   "linked": [],
   "access": "public",
-  "baseBranch": "V4",
+  "baseBranch": "V5",
```

Comment:
What does this do?

Reply:
https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#basebranch-git-branch-name

Will move to main

Reply:
In fact, this should only be used on releases so it's correct to point to V5